spec/features/user_creates_new_book_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+feature 'User creates a new book' do
+  scenario 'successfully' do
+    visit root_path
+
+    create_book("The Lord of the Rings: The Fellowship of the Ring", "J. R. R. Tolkien" )
+
+    expect(page).to have_css('.books li',
+                    text: 'The Lord of the Rings: The Fellowship of the Ring')
+    expect(page).to have_css('.books li', text: 'J. R. R. Tolkien')
```

Comment:
I wouldn't be too specific. `.books` is probably fine or if you want to be specific I would add a class to the list item like `book-row` or something and then you can use that in the `have_css` matcher